Completed sample assignment

MadsHaderup opened the pull request:
Every objective in the assignemnt has been completed, but the bonus task is without testing.

MadsHaderup committed:
initial commit, product importation, pagination and find by id

MadsHaderup committed:
complete CRUD implemented

MadsHaderup committed:
added some tests

MadsHaderup committed:
added some tests

MadsHaderup committed:
added requirements.txt

MadsHaderup committed:
cleaned up code a bit

MadsHaderup committed:
Update README.md

razius commented:
Hi Mads, 

Thank you for the submission. 

Can you add a gitignore file and clean-up the pull request of unnecessary files to make the pull request readable?

MadsHaderup committed:
Deleted pycache directory

MadsHaderup committed:
Deleted pycache directory

MadsHaderup committed:
Deleted venv directory

MadsHaderup committed:
Updated .gitignore

MadsHaderup commented:
Hi, 

I have added a gitignore file and removed unnecessary files.

MadsHaderup commented:
Hi Silviu,

Thanks for your email, I have added a gitignore file, and cleaned up the pull request! 

Best regards 
Mads

> Den 18. aug. 2022 kl. 13.24 skrev Silviu Tantos ***@***.***>:
> 
> 
> Hi Mads,
> 
> Thank you for the submission.
> 
> Can you add a gitignore file and clean-up the pull request of unnecessary files to make the pull request readable?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/unisport/sample/pull/82#issuecomment-1219375175>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKQLENYXRBB5KIH7352H6A3VZYMNLANCNFSM56XHVF6Q>.
> You are receiving this because you authored the thread.
